### DISCUSSION TITLE
Comply with the geoJSON specification's requirement to include the ID as a member of the Feature Object.

### DISCUSSION THREAD

**Discussion by DeraldDudley** [usdot-jpo-ode/jpo-wzdx#114] [General]
The Feature Object definition, in geoJson, states that commonly used IDs should be placed at the feature level (Not placed in the properties object).  To adhere to the geoJSON specification the WZD spec should require IDs be placed at the feature level.  (See 4th bullet below.)

Feature Object

A Feature object represents a spatially bounded thing.  Every Feature object is a GeoJSON object no matter where it occurs in a GeoJSON text.

 - A Feature object has a "type" member with the value "Feature".

  - A Feature object has a member with the name "geometry".  The value of the geometry member SHALL be either a Geometry object as defined above or, in the case that the Feature is unlocated, a JSON null value.

 - A Feature object has a member with the name "properties".  The value of the properties member is an object (any JSON object or a
      JSON null value).

 - If a Feature has a commonly used identifier, that identifier SHOULD be included as a member of the Feature object with the name "id", and the value of this member is either a JSON string or number.

**Comment by j-d-b** [comment]
Nice find. @DeraldDudley can you provide a link to this section of the offical GeoJSON spec?

**Reply by DeraldDudley** [reply]
[3.2 Feature Object](https://tools.ietf.org/html/rfc7946#section-3.2)

**Comment by j-d-b** [comment]
@DeraldDudley I think this is better suited as an issue, as it discusses a potential change to the specification. Can you create an issue for this? The content can be the same as here.

**Reply by DeraldDudley** [reply]
Sounds good. Will do.
